package.json

```diff
@@ -21,6 +21,7 @@
         "electron-updater": "^4.0.0",
         "flatbuffers": "latest",
         "handlebars": "latest",
+        "lodash": "^4.17.11",
```

Comment:
Remove loadash dependency.

Reply:
`filter` and `forEach` are available in JavaScript. It hasn't been used for any of the other formats. Your code is adding maintenance/dependency issues that didn't exist before.

Reply:
@lutzroeder I have made the removal of dependency. The usage was a bit broader: filter, map, find, includes. However, there are alternatives in vanilla JS.

Are you going to resolve discussions that are fixed or I should do that?